src/Tizen.NUI/src/public/Layouting/GridLayout.cs

```diff
@@ -304,7 +300,7 @@ public GridLayout()
         }
 
         /// <summary>
-        /// [Draft] Get/Set the number of columns in the GridLayout should have.
+        /// Get/Set the number of columns in the GridLayout should have.
```

Comment:
Grammar. There is inconsistency with the description of the Rows property "Get/Set the number of rows in the grid" which has the same logic. Also please use 3rd person to describe a property "Gets or sets the number..."

Reply:
DONE